docs/section23/Rule23-6.md

```diff
@@ -18,25 +18,32 @@
 **Function Calls:**  
 
 1. get_baseline_system_types()
-2. is_baseline_system_type()
+2. baseline_system_type_compare()
+
 
 **Applicability Checks:**  
 
-- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+- create a list of the target system types: target_system_types = [HVAC_SYS.SYS_6,HVAC_SYS.SYS_8]
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMI)`
 
-  - Check if B-RMR is modeled with at least one air-side system that is Type-6, 8, 8a, 6b, 8b, continue to rule logic: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-6", "SYS-8", "SYS-8A", "SYS-6B", "SYS-8B"]): CHECK_RULE_LOGIC`
+  - Check if B-RMI is modeled with at least one air-side system that is Type-6, 8, 8a, 6b, 8b, continue to rule logic: `if any(baseline_system_type_compare(system_type, target_sys_type, false) for system_type in baseline_hvac_system_dict.keys() for target_system_type in target_system_types): CHECK RULE LOGIC`
 
   - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
 
 ## Rule Logic:  
+- Create a list of the ids of the hvac systems that are one of the targeted system types: `eligible_systems = []`
+- For each baseline system type in the baseline_hvac_system_dict: `for baseline_system_type in baseline_hvac_system_dict:`
+  - check if the baseline_system_type is one of 6, 8, 8a, 6b, 8b: `if any(baseline_system_type_compare(baseline_system_type, target_system_type, false) for target_system_type in target_system_types):`
+    - add the hvac_ids for this system type to the list of eligible_systems: `eligible_systems += baseline_hvac_system_dict[baseline_system_type]`
+
 
-- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+- For each zone in B_RMR: `for zone_b in B_RMI...zones:`
```

Comment:
Why not use existing function get_dict_of_zones_and_terminal_units_served_by_hvac_sys to identify applicable terminals?